Documentation/development.md

```diff
@@ -0,0 +1,83 @@
+# CMO Development
+
+
+## Run CMO Locally
+### Assumptions
+* Should have a cluster running
+* Has setup Go development env
+
+### Build and Run CMO
+
+The `make` target `run-local` builds the `operator` and runs CMO locally
+
+```shell
+export KUBECONFIG=/path/to/kubeconfig/file
+make run-local
+```
+
+### Fix jsonnet
```

Comment:
I think we need a more specific heading here. Maybe something like "working with jsonnet" and a subsection for different operations. I find Fix jsonnet is a bit blunt and unspecific.